docs/connect/index.md

```
@@ -164,6 +116,60 @@ CrateDB drivers and adapters for supported programming languages, frameworks, an
+::::{grid-item-card} {material-outlined}`article;1.5em` Documentation
+:columns: 3
+- [HTTP interface]
+- [PostgreSQL interface]
+::::
+
+::::{grid-item-card} {material-outlined}`link;1.5em` Related
+:columns: 3
+- [Authentication]
+- [SQL query syntax]
+- [Bulk operations]
+- [BLOB support][CrateDB BLOBs]
```

markdownlint-cli2 (0.18.1)

Reference links and images should use a label that is defined
Missing link or image reference definition: "cratedb blobs"

(MD052, reference-links-images)